service/controller/v14/version_bundle.go

```diff
@@ -4,6 +4,8 @@ import (
 	"github.com/giantswarm/versionbundle"
 )
 
+const CoreosVersion = "2303.4.0"
```

Comment:
Minor note: `key` package could be better place in terms of "where is this configured?". That's where it is in `kvm-operator` as well: https://github.com/giantswarm/kvm-operator/blob/master/service/controller/key/key.go#L56

Reply:
makes sense

Reply:
done